pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1510,18 +1515,67 @@ func getNetworkProviderConfig(ctx context.Context, configManager ConfigManager,
 		return nil, errorUtil.Wrap(err, "failed to unmarshal aws vpc create config")
 	}
 
-	if vpcCreateConfig.CidrBlock == nil {
-		return nil, errorUtil.New(fmt.Sprintf("CidrBlock in _network strategy tier %s can not be nil", tier))
+	// if the config map is found and the _network block contains an entry, that is returned for use in the network creation
+	if vpcCreateConfig != nil && vpcCreateConfig.CidrBlock != nil && *vpcCreateConfig.CidrBlock != "" {
```

Comment:
I think we can get rid of the first nil check, as we dont dereference the `vpcCreateConfig` object. We will catch the nil pointer in `vpcCreateConfig.CidrBlock != nil`